src/test/ui/macros/macro-in-expression-context-2.stderr

```diff
@@ -5,7 +5,15 @@ LL | macro_rules! empty { () => () }
    |                            -- in this macro arm
 ...
 LL |         _ => { empty!() }
-   |                ^^^^^^^^ expected expression
+   |                ^^^^^^^^
+   |                |
+   |                expected expression
+   |                this macro call doesn't expand to an expression, but it does to a statement
```

Comment:
"but it does to a statement" sounds like it could be confusing to people (mainly ESL). Maybe we can add a bit of redundancy to the copy or make it a `note` so that we don't worry about its length? Maybe:

```
  = note: the macro call doesn't expand to an expression, but it can expand to a statement
```

Reply:
I'll make it a note. :)